apps/nameguard.io/src/app/docs/displayed-name/page.tsx

```diff
@@ -0,0 +1,152 @@
+"use client";
+
+import { ENSName, buildENSName } from "@namehash/ens-utils";
+import { DisplayedName } from "@namehash/nameguard-react";
+
+export default function DisplayedNamePage() {
```

Comment:
Ideally we move all these examples to the storybook. But until we do that please ensure the `/docs` page includes a link to each of these types of pages you've created. Thanks.